services/121-service/src/payments/fsp-integration/intersolve-visa/dto/internal/create-physical-card.dto.ts

```diff
@@ -0,0 +1,12 @@
+// TODO: Renamed from IntersolveCreateCustomerDto, since this DTO is only used in IntersolveVisaModule, so the context is clearly Intersolve. Is that ok?
+export class CreatePhysicalCardDto {
+  public readonly tokenCode: string;
+  public readonly name: string;
+  public readonly addressStreet: string;
+  public readonly addressHouseNumber: string;
+  public readonly addressHouseNumberAddition?: string;
+  public readonly addressPostalCode: string;
+  public readonly addressCity: string;
+  public readonly phoneNumber: string;
+  public readonly coverLetterCode: string;
+}
```

Comment:
I see that this and the previous Dto have a lot of information in common. Would it make sense to share this info?

```ts
export interface Customer {
  readonly name: string;
  readonly addressStreet: string;
  readonly addressHouseNumber: string;
  readonly addressHouseNumberAddition?: string;
  readonly addressPostalCode: string;
  readonly addressCity: string;
  readonly phoneNumber: string;
}

export interface CreatePhysicalCardDto extends Customer {
  readonly tokenCode: string;
  readonly coverLetterCode: string;
}
```

Reply:
YEs it would! Peter and I also came up with this idea already, but did not refacor it yet. I do not see a "commit" button on this suggestion, so will add it "manually".

Reply:
Will do it afer merging PR to feat.